rbi/stdlib/set.rbi

```diff
@@ -992,8 +992,8 @@ end
 #
 # for pack.c
 class Array
-  sig {returns(T::Set[T.untyped])}
-  def to_set; end
+  sig { params(blk: T.nilable(T.proc.params(arg0: T.untyped).void)).returns(T::Set[T.untyped]) }
+  def to_set(&blk); end
```

Comment:
Btw, I am not even sure if `Array#to_set` is actually a thing. Looking at the Ruby `set` code, I [see](https://github.com/ruby/ruby/blob/master/lib/set.rb#L19):
> The method +to_set+ is added to Enumerable for convenience.

and only [a definition for `Enumerable#to_set`](https://github.com/ruby/ruby/blob/master/lib/set.rb#L684-L690) in the file.

Is this something that can be removed? @jez was this needed for some internal operation?

Reply:
I don't know about the historical context, it's possible it's incorrect.

Reply:
Yeah, I think it is best to remove the `Array` specialization, since the `Enumerable` one does the right thing. I believe it even passes the tests @RyanBrushett has added in this PR.

@RyanBrushett can you open a new PR with this specialization removed please? 🙏

Reply:
#4087

Reply:
You are too quick 😂

Reply:
The tests I added will need to be removed/changed but otherwise that should work yep 👍